pkg/proto/time/prototime.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright © 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prototime
+
+import (
+	"time"
+
+	google_duration "github.com/golang/protobuf/ptypes/duration"
+	google_protobuf "github.com/golang/protobuf/ptypes/timestamp"
+)
+
+// TimeToTimestamp converts a go Time to a protobuf Timestamp.
+func TimeToTimestamp(t time.Time) *google_protobuf.Timestamp {
```

Comment:
I think ptypes has most of these functions already. We use them in OpenStorage [API](https://github.com/libopenstorage/openstorage/blob/8be8bb8b90578ff2cfa84e1e90958df61f64dc9b/api/api.go#L912)

Reply:
These 2 files are copied from libopenstorage.